### ISSUE TITLE
volume-definition set-size runs drbdadm resize on diskless nodes

### ISSUE THREAD

**Issue by jiri-lunacek** [LINBIT/linstor-server#48]
Hi.

I have a 3 node cluster, one being diskless like so:
```
LINSTOR ==> storage-pool list
╭───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
┊ StoragePool ┊ Node                  ┊ Driver         ┊ PoolName        ┊ FreeCapacity ┊ TotalCapacity ┊ SupportsSnapshots ┊
╞┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄╡
┊ drbdpool    ┊ arbiter1.hosting90.cz ┊ DisklessDriver ┊                 ┊              ┊               ┊ false             ┊
┊ drbdpool    ┊ drbd1.hosting90.cz    ┊ LvmThinDriver  ┊ centos/drbdpool ┊    18.00 TiB ┊     18.00 TiB ┊ true              ┊
┊ drbdpool    ┊ drbd2.hosting90.cz    ┊ LvmThinDriver  ┊ centos/drbdpool ┊    18.00 TiB ┊     18.00 TiB ┊ true              ┊
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

When I tried to resize a volume:
`volume-definition set-size target123459 0 200GiB`

linstor tried to run drbdadm resize on the diskless node:
```
drbdadm -vvv resize target123459/0
The resize command requires a local disk, but the configuration gives none
```

The volume is now stuck in _resizing_ status
`┊ target123459 ┊ 0        ┊ 1002        ┊ 200 GiB ┊ resizing ┊`

**Comment by rp-** [comment]
Thanks for the report, we are testing a fix internally right now and will most likely release a new version very soon.

**Comment by rp-** [comment]
issue should be fixed in release 0.9.3
